webpack-demo/src/tasks.js

```diff
@@ -0,0 +1,30 @@
+export const saveTasks = (tasks) => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+// Define the functions for adding tasks
+export const addTask = (description, tasks) => {
+  const task = {
+    description,
+    completed: false,
+    index: tasks.length,
+  };
+  tasks.push(task);
+  saveTasks(tasks);
+  return task;
+};
+
+// Function for deleting or removing tasks
+export const deleteTask = (index, tasks) => {
+  tasks.splice(index, 1);
+  for (let i = index; i < tasks.length; i += 1) {
```

Comment:
- [ ]as requested by the previous reviewer kindly consider fixing this line  it has some logic error cause as i delete a list in the array the list is disappearing with it current index instead of decrementing and the requirement says below 👇 
> Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.

here is a hint on how to fix that line `  for (let i = index; i < tasks.length; i --) {`

adding the above change will make your app behave as expected